src/storage/heap_file.c

```diff
@@ -7820,10 +7819,102 @@ heap_mvcc_get_for_delete (THREAD_ENTRY * thread_p, OID * oid, OID * class_oid, R
 			  HEAP_SCANCACHE * scan_cache, int ispeeking, int old_chn, MVCC_REEV_DATA * mvcc_reev_data,
 			  NON_EXISTENT_HANDLING non_ex_handling_type)
 {
+  HEAP_GET_CONTEXT context;
+  SCAN_CODE scan;
+
```

Comment:
@andrei14vl 

It seems there's no call to `heap_mvc_get_for_delete`. 
Are you going to replace it with `heap_mvcc_get_for_delete_new`?